hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/account/MirrorEvmContractAliases.java

```diff
@@ -44,7 +45,7 @@ public Address resolveForEvm(Address addressOrAlias) {
         final var entityOptional = mirrorEntityAccess.findEntity(addressOrAlias);
 
         if (entityOptional.isEmpty()) {
-            throw new InvalidParametersException("No such contract or token");
+            throw new EntityNotFoundException("No such contract or token");
```

Comment:
We should print out the hex address so users know which address is missing. Same comment for the other throw statement but change it to `"Not a contract or token: " + addressOrAlias`.
```suggestion
            throw new EntityNotFoundException("No such contract or token: " + addressOrAlias);
```